Tests/test_map_ui.gd

```diff
@@ -1,22 +1,22 @@
-extends GutTest
+extends TestMapBase
 
 var map_ui_scene: PackedScene = load("res://#Scenes/MapUI.tscn")
 var map_ui: MapUI
 var test_map_array: Array[int] = [1, 3, 5, 3, 1]
 
-func before_all():
+func before_all() -> void:
```

Comment:
the `before_all` is already defined in `TestMapBase`
You just need to do an override to add `map_ui = map_ui_scene.instantiate` since this is not in the base file

edit well technically we recreate the map each_time with a before each, which prevents adverse effects from previous tests

Reply:
I ended up removing out before_all() and put everything in before_each(), thanks for the suggestion